docs/integrations/github/discovery.md

```diff
@@ -53,6 +53,51 @@ And then add the entity provider to your catalog builder:
   }
 ```
 
+## Installation with Events Support
+
+Please follow the installation instructions at
+
+- https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md
+- https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-github/README.md
+
+Additionally, you need to decide how you want to receive events from external sources like
+
+- [via HTTP endpoint](https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md)
```

Comment:
I believe this will be the method used by most users. Would it be worth adding a few lines describing how to configure the webhook in github?